### PR TITLE
Fixes to animation

### PR DESCRIPTION
**Pull request by bcasselsND**
My changes for making mins and maxes more flexible and organized,
missed a couple of places. Fix those.

Put elevationWindow into animationEngine, as activeWindow is. This
fixes some inconsistencies.

Put initial values for activeWindow and elevationWindow as an argument
to the AnimationEngine constructor. Take those values, and shove them
in the playbar DOM, overriding anything in the HTML. [I'm afraid to
take them out of the HTML, but maybe someday....]

Baby steps at cleaning up the spaghetti mess of potree code.

### CODE REVIEW

**Comment by bcasselsND** [comment]
@NickRizzo-droid Please check my commit, from after your review. Addresses problems with wheel scrolling in the scroll bar.

File: common/playbar.js [hunk 1]
```diff
@@ -120,37 +134,27 @@ export function createPlaybar () {
 
     }
 
-    playbarhtml.find("#myRange").on('input', updateSlider);
+    playbarhtml.find("#myRange").on('input', function() {
+      updateSlider();
+    });
```
Comment (NickRizzo-droid) [inline common/playbar.js:139]: Why not '=> notation'? What standard/style are we using?

Comment (bcasselsND) [reply common/playbar.js:139]: Why I did it this way is that I am restoring code that I had removed. Really, this wants to be remodularized a bit. I'll do that now, and clean up a little tiny mess.

Comment (NickRizzo-droid) [reply common/playbar.js:139]: okay sounds good! Ping me when I should look again

Comment (NickRizzo-droid) [reply common/playbar.js]: Changes look much better

**Comment by NickRizzo-droid** [comment]
Looks good to go

**Pull request merged**